docs/examples.md

```diff
@@ -0,0 +1,150 @@
+# Examples
+
+This Example has been taken with a Ledger nano.
+
+```
+./hwi.py enumerate
+[{"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@0/IOUSBHostHIDDevice@14200000,0", "serial_number": "0001"}, {"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@1/IOUSBHostHIDDevice@14200000,1", "serial_number": "0001"}]```
```

Comment:
That's... interesting. On Ubuntu, the device paths are much much shorter. I see things like `0003:0001:00` for the path. I guess then that you should also say what OS you are using as the paths may be different depending on OS.

Reply:
Ok no problem. i also have to clean up the modified ledger.md.